students/19th/westagram/style/login.css

```
@@ -0,0 +1,57 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+*{box-sizing: border-box;}
```

```suggestion
*{box-sizing: border-box;}
```
이 부분은 comment.css에 모아서 하면 좋을 것 같아요!